Frameworks/Foundation/NSCache.mm

```diff
@@ -286,7 +286,7 @@ - (void)_evictEntriesOverCharge {
 
     auto rit = _cacheEntries.end();
     do {
-        _NSCacheEntry* entry = (id)*--rit;
+        _NSCacheEntry* entry = (id) * --rit;
```

Comment:
I'm just going to remove this file from the final commit

---
In reply to: [70347524](https://github.com/Microsoft/WinObjC/pull/634#discussion_r70347524) [](ancestors = 70347524)